tests/Microsoft.VisualStudio.Editors.UnitTests/Microsoft.VisualStudio.Editors.UnitTests.vbproj

```diff
@@ -0,0 +1,40 @@
+﻿<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\import\VisualStudio.props" />
+  <PropertyGroup>
+    <RootNamespace>Microsoft.VisualStudio</RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="My Project\**" />
+    <EmbeddedResource Remove="My Project\**" />
+    <None Remove="My Project\**" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Remove="Resource1.resx" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Remove="explorer.ico" />
+    <None Remove="img0.jpg" />
```

Comment:
remove unnecessary file reference

Reply:
This is the handiwork of Visual Studio not cleaning up the file properly. I've cleaned it myself now.